frame.test.js

```diff
@@ -0,0 +1,50 @@
+const Frame = require("./frame");
+
+describe("class: Frame", () => {
```

Comment:
Tests are really well-structured and easy to follow. Your nested describe blocks make it really clear!